brayns/common/scene/Scene.cpp

```diff
@@ -145,28 +173,25 @@ size_t Scene::addModel(ModelDescriptorPtr model)
 
 void Scene::removeModel(const size_t id)
 {
+    bool removed = false;
```

Comment:
Is this a fix some other issue? Or is this required for the clip planes?

Reply:
No, but Daniel requested not to call markModified if nothing changed for clip planes, so I propagated the change here.